### PR TITLE
BD-974: Add simple survey use cases

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

**Description of Change:**
> I'm adding sample use case to the simple survey doc

Closes #**BD-974**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

**Comment by davidbielik** [comment]
@bre-fitzgerald I'm not sure why the descriptions aren't showing:
image

**Comment by davidbielik** [comment]
> @bre-fitzgerald I'm not sure why the descriptions aren't showing:
> image

- nevermind! it's there now

**Review by davidbielik — APPROVED** [review]
Thank you!!

File: _docs/_user_guide/message_building_by_channel/in-app_messages/templates/simple_survey.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 nav_title: "Simple Survey"
 page_order: 1.5
 page_type: reference
-hidden: true
```
Comment (KellieHawks) [inline _docs/_user_guide/message_building_by_channel/in-app_messages/templates/simple_survey.md:5]: @bre-fitzgerald Are you sure you want this listed in docs yet? If so, can you add a blank config for the templates drop-down. 
Screen Shot 2021-06-30 at 3 52 50 PM

**Review by KellieHawks — CHANGES_REQUESTED** [review]

File: _docs/_user_guide/message_building_by_channel/in-app_messages/templates/simple_survey.md (hunk 1 above)
Comment (davidbielik) [reply _docs/_user_guide/message_building_by_channel/in-app_messages/templates/simple_survey.md:5]: hey @KellieHawks i removed the hidden field. Do we need an overview page, considering it's only one template at the moment?

Comment (KellieHawks) [reply _docs/_user_guide/message_building_by_channel/in-app_messages/templates/simple_survey.md:5]: @davidbielik Oh gotcha, it doesn't need an overview page but it does need a blank_config page so the word "templates" displays as "Templates". I'm sure Bre will add it in 👍🏻

Comment (bre-fitzgerald) [reply _docs/_user_guide/message_building_by_channel/in-app_messages/templates/simple_survey.md:5]: @davidbielik @KellieHawks Updated the templates page to be config_only 👍

**Review by KellieHawks — APPROVED** [review]
Looks great!

**Pull request merged**